app/slimming/slimming.js

```diff
@@ -1,11 +1,30 @@
 'use strict';
 app.controller('GOSlimCtrl', function($scope, $location, $q,
-  hardCodedDataService, presetsService, termService, basketService,
+  hardCodedDataService, presetsService, $document, termService, basketService,
```

Comment:
Ok, I can see the hardcoded stuff was there already so I will create a new issue for that.